package.json

```diff
@@ -36,7 +36,7 @@
     "js-priority-queue": "^0.1.5",
     "jszip": "^3.1.3",
     "proj4": "^2.4.3",
-    "three": "^0.84.0",
+    "three": "^0.85.0",
```

Comment:
I can give it a try and see if it works correctly; r86 it was released yesterday :)